### PR TITLE
using different storage class name

### PR DESCRIPTION
**Pull request by osherdp**
/cc @osherdp 
/test pj-rehearse

### CODE REVIEW

File: ci-operator/step-registry/baremetalds/assisted/operator/setup/baremetalds-assisted-operator-setup-commands.sh [hunk 1]
```diff
@@ -119,14 +119,14 @@ metadata:
  name: agent
```
Comment (djzager) [inline ci-operator/step-registry/baremetalds/assisted/operator/setup/baremetalds-assisted-operator-setup-commands.sh:119]: Not super important but the AgentServiceConfig isn't namespaced so you can drop the namespace.

Comment (osherdp) [reply ci-operator/step-registry/baremetalds/assisted/operator/setup/baremetalds-assisted-operator-setup-commands.sh:119]: ack

**Pull request closed without merging**